src/traces/parcoords/parcoords.js

```diff
@@ -348,6 +350,17 @@ function parcoordsInteractionState() {
     };
 }
 
+function calcTilt(angle) {
```

Comment:
How does this look like for multi-line labels? 

Cartesian labels have this:

https://github.com/plotly/plotly.js/blob/06901623e988b7ad036207a905a4a032da186357/src/plots/cartesian/axes.js#L2482-L2489

Reply:
See example https://rreusser.github.io/plotly-mock-viewer/#bar_multiline_labels

Reply:
For the moment the labels are positioned using the first line.
This is pretty cool! I may give that a try on this PR.